packages/contracts/src/deploy-config.ts

```diff
@@ -238,9 +233,6 @@ export const deployConfigSpec: {
     type: 'number',
     default: 1,
   },
-  finalSystemOwner: {
-    type: 'address',
-  },
   l1StartingBlockTag: {
     type: 'string',
   },
```

Comment:
Could you add `securityCouncilTokenOwner`, `securityCouncilNumConfirmationRequired`, `securityCouncilOwners`, `governorVotingDelayBlocks`, `governorVotingPeriodBlocks`, `governorProposalThreshold`, `governorVotesQuorumFractionPercent`, `timeLockMinDelaySeconds` to `DeployConfig`? I think these variables are missing.

Reply:
Okay, But it's impossible to add `securityCouncilOwners` because the `deploy-config` interface does not support `string[]` type.

Reply:
Considering this situation and thinking about it, it makes sense that this value should be removed in the future, so 
I won't add it.